### PR TITLE
changes after feedback

### PR DESCRIPTION
**Pull request by mrmicjam**
Just some changes to the documentation and defaults after some feedback

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
 # A Simple SAML Identity Provider (IdP) for Testing SSO Interfaces
 
+## Using test-saml-idp at Sitka
+
+* If using locally against the existing okta dev IDP, ask Renaud or Micah for the test certificates and save to this folder.
```
Comment (run-o) [inline README.md]: Should we just commit the certs to the repo? I don't see any security issue since these are just fake certs for local testing.

Comment (mrmicjam) [reply README.md]: Nope!  It just feels a little strange, but my logical mind is telling me its ok in this case haha.  I'll push 'em up

Comment (mrmicjam) [reply README.md]: Added certs and updates some readme content accordingly with https://github.com/trustsitka/test-saml-idp/pull/2/commits/34bd1125603a8e8fbcdbbb24039f5dc1a9179c36

**Review by run-o — APPROVED** [review]
Thanks!

**Pull request merged**